tendrl/node_agent/monitoring/collectd/templates/gluster/tendrl_glusterfs_brick_utilization.jinja

```diff
@@ -7,5 +7,6 @@
         integration_id "{{integration_id}}"
         graphite_host "{{ graphite_host }}"
         graphite_port "{{ graphite_port }}"
+        peer_name "{{ hostname }}"
```

Comment:
The hostname is passed to all plugins as in https://github.com/Tendrl/commons/blob/master/tendrl/commons/objects/cluster/atoms/configure_monitoring/__init__.py#L92